app/views/products/index.html.erb

```diff
@@ -4,7 +4,7 @@
   <th> Description </th>
   <% @products.each do |product|  %>
     <tr>
-    <td><%= product.name%></td>
+    <td><%= link_to product.name, product %></td>
     <td><%= product.description%></td>
```

Comment:
identation

Reply:
This line needs a space `<td><%= product.description%></td>` here `...description %>`

Reply:
Ok. 👍